packages/react-core/src/components/ContextSelector/ContextSelector.tsx

```diff
@@ -147,13 +147,7 @@ export class ContextSelector extends React.Component<ContextSelectorProps, { oui
     const screenReaderLabelId = `pf-context-selector-label-id-${uniqueId}`;
 
     const menuContainer = (
-      <div
-        className={css(styles.contextSelectorMenu)}
-        // This removes the `position: absolute`styling from the `.pf-c-context-selector__menu`
-        // allowing the menu to flip correctly
-        {...(isFlipEnabled && { style: { position: 'revert' } })}
-        id={uniqueId}
-      >
+      <div className={css(styles.contextSelectorMenu, isFlipEnabled && styles.modifiers.static)} id={uniqueId}>
```

Comment:
Looks like this also needs to check for `menuAppendTo !== 'inline'` since that's the default. Currently menu is pushing content down when it's expanded.

Reply:
Ah good catch! Latest push includes the similar `isStatic` lines as the other components.